qa/tasks/cephfs/test_nfs.py

```diff
@@ -83,74 +105,116 @@ def _test_list_cluster(self, empty=False):
         self.assertEqual(cluster_id, nfs_output.strip())
 
     def _create_export(self, export_id, create_fs=False, extra_cmd=None):
+        '''
+        This tests creation of a single export.
+        :param export_id: Denotes export number
+        :param create_fs: If false filesytem exists. Otherwise create it.
+        :param extra_cmd: List of extra arguments for creating export.
+        '''
         if create_fs:
             self._cmd('fs', 'volume', 'create', self.fs_name)
         export_cmd = ['nfs', 'export', 'create', 'cephfs', self.fs_name, self.cluster_id]
         if isinstance(extra_cmd, list):
             export_cmd.extend(extra_cmd)
         else:
             export_cmd.append(self.pseudo_path)
-
+        # Runs the nfs export create command
         self._cmd(*export_cmd)
         res = self._sys_cmd(['rados', '-p', 'nfs-ganesha', '-N', self.cluster_id, 'get',
                              'export-%s' % (export_id), '-'])
+        # Check if export object is created
         if res == b'':
-            raise RuntimeError("Export cannot be created")
+            self.fail("Export cannot be created")
 
     def _create_default_export(self):
-            self._test_create_cluster()
-            self._create_export(export_id='1', create_fs=True)
+        '''
+        It deploys a single nfs cluster and creates export with default options.
```

Comment:
Here and everywhere else, you can omit 'It' and 'This' and be consistent throughout in the descriptions.
This can just be,
```
Deploys a single nfs cluster and creates export with default options.
```

Reply:
Done